ecosystem/sep-0041.md

```diff
@@ -200,6 +202,12 @@ The event has topics:
 The event has data:
 
 - `i128` the amount transferred.
+- or `map` containing entries with `Symbol` keys, and values defined as
+  - `amount: i128` the amount minted.
+  - `to_muxed_id: Option<u64 | String | BytesN<32>>` void, or entry absent, if
+    no muxed ID, or the u64 muxed ID, or for historical legacy reasons, the
+    string or bytes memo for Stellar assets.
```

Comment:
> the string or bytes memo for Stellar assets.

I don't understand how the string or bytes memo relates to stellar assets. A string or bytes type will only be emitted when the text or hash memo types are used on the transaction memo right? Should this say "the string or bytes memo from the transaction" instead?

Reply:
I don't have a too strong opinion, but if we look at this from the perspective of someone who wants to consume the events, they shouldn't need to worry about whether the to_muxed_id coming from. Pointing out that this exists for legacy reasons to support specifically Stellar assets hopefully clears up things for the token implementers. At least that's my interpretation. If we want to talk about memos, we could, but we need to be precise there then (i.e. point out that the memo comes from the 'classic' transactions).